ERCS/erc-draft_payout_race.md

```diff
@@ -0,0 +1,222 @@
+---
+eip: TBD
```

Comment:
```suggestion
eip: 8017
```
Assigning next sequential EIP/ERC/RIP number.
Numbers are assigned by editors & associates.

Please also update the filename.